scripts/githooks/check-file-format.sh

```diff
@@ -75,8 +68,32 @@ function is-arg-true() {
 
 # ==============================================================================
 
+check=${check:-working-tree-changes}
+
+case $check in
+    "all")
+        filter="git ls-files"
+        ;;
+    "staged-changes")
+        filter="git diff --diff-filter=ACMRT --name-only --cached"
+        ;;
+    "working-tree-changes")
+        filter="git diff --diff-filter=ACMRT --name-only"
+        ;;
+    "branch")
+        filter="git diff --diff-filter=ACMRT --name-only ${BRANCH_NAME:-origin/main}"
+        ;;
+    *)
+        echo "Unrecognised check mode: $check" >&2 && exit 1
+        ;;
+esac
+
+echo $check
```

Comment:
Do we need the `echo`s?

Reply:
...no, we don't.  That shouldn't have been left in.